src/content-helper/dashboard-widget/dashboard-widget.scss

```diff
@@ -61,7 +85,7 @@
 
 	.parsely-top-post-thumbnail {
 		height: 46px;
-		width: 46px;
+		min-width: 46px;
```

Comment:
Out of curiosity: Did you run into an issue and changed that? Wondering if this could result in different width images, which might make the widget look disarranged.

Reply:
I did change it to min-width to make all the thumbnails have the same width, which was not happening when mixing vertical and horizontal images,  for example:

![image](https://github.com/Parsely/wp-parsely/assets/5804923/d6b1c3d4-6c68-458e-9af9-4b7c103a1c41)

With this change, they become aligned, and square, as intended (I think?):

![image](https://github.com/Parsely/wp-parsely/assets/5804923/3fcf4770-7c45-4a24-b99e-9830b2f725ae)

Reply:
Ah, excellent!